conan/tools/sbom/cyclonedx.py

```diff
@@ -10,6 +10,9 @@ def cyclonedx_1_4(graph, add_build=False, add_tests=False, **kwargs):
 
     has_special_root_node = not (getattr(graph.root.ref, "name", False) and getattr(graph.root.ref, "version", False) and getattr(graph.root.ref, "revision", False))
     special_id = str(uuid.uuid4())
+
+    name_default = graph.root.ref.name if bool(getattr(graph.root.ref, "name", False)) else "conan-sbom"
```

Comment:
A ``if bool(something)`` is basically the same as ``if something``, or is this an exception?
Then you can use short-circuit ``or``
```suggestion
    name_default = getattr(graph.root.ref, "name", False) or "conan-sbom"
```